README.md

```diff
@@ -277,6 +277,20 @@ $ go-licenses check \
 Note that dependencies from the ignored packages are still resolved and checked.
 This flag makes effect to `check`, `report` and `save` commands.
 
+### Include testing packages
+
+Use the `--include_tests` global flag to include packages only imported by testing code.
+For example, those packages could be testing libraries/frameworks.
```

Comment:
For example, testing libraries or frameworks.

Reply:
That's not a complete sentence though. I merged the two sentences to make it more concise.